File: app/src/main/java/com/alex/materialdiary/sys/common/models/diary_day/Mark.java
package com.alex.materialdiary.sys.common.models.diary_day;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Mark {

    @SerializedName("SYS_GUID")
    @Expose
    private String sysGuid;
    @SerializedName("LONG_NAME")
    @Expose
    private String longName;
    @SerializedName("SHORT_NAME")
    @Expose
    private String shortName;
    @SerializedName("VALUE")
    @Expose
    private Integer value;

    public String getSysGuid() {
        return sysGuid;
    }

    public void setSysGuid(String sysGuid) {
        this.sysGuid = sysGuid;
    }

    public String getLongName() {
        return longName;
    }

    public void setLongName(String longName) {
        this.longName = longName;
    }

    public String getShortName() {
        return shortName;
    }

    public void setShortName(String shortName) {
        this.shortName = shortName;
    }

    public Integer getValue() {
        return value;
    }

    public void setValue(Integer value) {
        this.value = value;
    }

}

File: app/src/main/java/com/alex/materialdiary/sys/common/models/diary_day/Mark.kt
package com.alex.materialdiary.sys.common.models.diary_day

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import kotlinx.parcelize.Parcelize

@Parcelize
class Mark : Parcelable {
    @SerializedName("SYS_GUID")
    @Expose
    var sysGuid: String? = null

    @SerializedName("LONG_NAME")
    @Expose
    var longName: String? = null

    @SerializedName("SHORT_NAME")
    @Expose
    var shortName: String? = null

    @SerializedName("VALUE")
    @Expose
    var value: Int? = null
}